test-reminder-email.sh

```
@@ -0,0 +1,63 @@
+
+curl -s -X GET "http://$HOST:$PORT/api/reminders/test/$EMAIL" | jq .
+
+# 2. Test the POST endpoint with a complete request
+echo -e "\n${BLUE}2. Testing POST endpoint with a complete request...${NC}"
+
+curl -s -X POST "http://$HOST:$PORT/api/reminders/send" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "plantName": "Script Test Plant",
+    "reminderType": "Water",
+    "reminderDate": "'$(date -I)'",
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)